recipes/imbalanced-learn/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "imbalanced-learn" %}
+{% set version = "0.3.1" %}
+{% set sha256 = "5dfd6773b5f5e067ebcc6b032907ce721227a2f2eb0b684f3b169c65a10b5257" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - scikit-learn 0.19.*
```

Comment:
Is it linking to `scikit-learn` or is there some other reason this is needed at build time?

Reply:
It is linked to `scikit-learn`.

Reply:
As in, it is compiling shared libraries that load shared libraries from `scikit-learn`?

Reply:
If it isn't building shared libraries, we should be able to drop `scikit-learn` from `requirements/build`.

Reply:
Let say this is due to some project template. We used scikit-learn project template which check the packages version which will be needed at run time during install([see here](https://github.com/scikit-learn-contrib/imbalanced-learn/blob/master/imblearn/version.py#L82)). Therefore, build and run packages are the same even if we don't need them.

I am aware that this is in fact not useful and now that I am aware of this issue, I am going to change and update the setup for the next release to avoid a such heavy thing for no reason.

Are you fine with creating the feedstock and let me update the recipe for the next release.

Reply:
Ah ok. So this is run from `setup.py` somewhere?

Should we also be specifying `numpy` and `scipy` as is done in that file?

Reply:
> Ah ok. So this is run from setup.py somewhere?

yes from [here](https://github.com/scikit-learn-contrib/imbalanced-learn/blob/master/setup.py#L64)

> Should we also be specifying numpy and scipy as is done in that file?

No, because we are based on the same requirements than latest release of scikit-learn where numpy and scipy versions are already defined.

Reply:
Ok. I made the changes in this regard.